torchvision/prototype/transforms/_augment.py

```diff
@@ -190,6 +190,90 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
             return inpt
 
 
+def flatten_and_extract_data(
```

Comment:
Perhaps it's too early but worth benchmarking that this doesn't introduce a significant speed regression on SimpleCopyPaste.